dbusmock/templates/ofono.py

```diff
@@ -79,6 +94,7 @@ def AddModem(self, name, properties):
                        'Manufacturer': dbus.String('Fakesys', variant_level=1),
                        'Model': dbus.String('Mock Modem', variant_level=1),
                        'Revision': dbus.String('0815.42', variant_level=1),
+                       'Serial': dbus.String(newSimSerial(), variant_level=1),
```

Comment:
So `newSimSerial()` is only being called once, ever. Why the overhead with the function and counter instead of just setting it to a static value and letting the test case drive/set it? Mocks shouldn't be doing unnecessary things by themselves, especially not if the "real thing" doesn't do the same. And I doubt that the serial number changes?